src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -4,55 +4,43 @@
 
 package frc.team2412.robot;
 
-import static java.lang.Thread.sleep;
-
-import frc.team2412.robot.commands.shooter.ShooterResetEncodersCommand;
 import static frc.team2412.robot.Subsystems.SubsystemConstants.*;
+import static java.lang.Thread.sleep;
 
-import frc.team2412.robot.sim.PhysicsSim;
-import frc.team2412.robot.sim.SparkMaxSimProfile.SparkMaxConstants;
-import frc.team2412.robot.sim.TalonFXSimProfile.TalonFXConstants;
+import java.io.IOException;
+import java.net.NetworkInterface;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
 
 import org.frcteam2910.common.math.RigidTransform2;
 import org.frcteam2910.common.robot.UpdateManager;
 
 import edu.wpi.first.hal.simulation.DriverStationDataJNI;
 import edu.wpi.first.wpilibj.DriverStation;
-import edu.wpi.first.wpilibj.PowerDistribution;
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj.smartdashboard.Field2d;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
 import frc.team2412.robot.Subsystems.SubsystemConstants;
```

Comment:
By the way, we're importing `SubsystemConstants` and statically importing all of its members. Not a you thing, just something that should be fixed at some point by somebody.